jproject-fr/easy-eat
# #K610: Add new address on step 2 should be clickable

Modified /poc/shippingaddress.vue and replaced the button by a DialogDisplay that allows the user to add a new address. +Refactored the component import

TO TEST:
- Go to /poc/shippingaddress
-  Click on add new address 
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/47e1a5ea-2363-4e4b-8626-77c2bf2bc437)
 - This form should appear:
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/2778cb1e-38d9-4edb-8223-28610a1f6f37)
